tests/test_check.py

```diff
@@ -42,7 +42,7 @@ def mock_parse_log(log):
         m_open = mock_open()
         open_name = 'util.open'
         with patch(open_name, m_open, create=True):
-            check.check_regression('pkgdir', False)
+            check.check_regression('pkgdir', False, -1)
```

Comment:
I would prefer for the new argument to be `1`, since the `-` is treated as a hyphen in `check_regression`, and it looks like a "negative 1" here at the call site.

Reply:
ignore.. I misread the code.